master/internal/command/command_service.go

```diff
@@ -158,6 +158,7 @@ func (cs *CommandService) listByType(
 	reqUserIDs []int32,
 	cmdType model.TaskType,
 	workspaceID int32,
+	reqIDs *[]string,
```

Comment:
nit: I think I prefer reqIDs being not a pointer, to match the style of reqUsers

Reply:
It would be nice to be consistent, but I wanted to make it optional and use `nil` for non command task